tox.ini

```diff
@@ -3,8 +3,17 @@
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
 
+[flake8]
+# NOTE: This is kept in line with Black
+#       See: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length
+max-line-length = 88
+
 [tox]
-envlist = py{35,36,37,38,py3}-pytest{50,51,52,53,54},
+distshare = {homedir}/.tox/distshare
```

Comment:
According to [docs](https://tox.readthedocs.io/en/latest/config.html#conf-distshare), this seems to be the default, or am I wrong?